src/aave-v3/SupplyVault.sol

```diff
@@ -126,37 +122,40 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
         view
         returns (address[] memory rewardTokens, uint256[] memory unclaimedAmounts)
     {
-        uint256 supply = totalSupply();
-        if (supply > 0) {
-            uint256[] memory claimableAmounts;
+        uint256[] memory claimableAmounts;
 
-            {
-                address[] memory poolTokens = new address[](1);
-                poolTokens[0] = poolToken;
+        {
+            address[] memory poolTokens = new address[](1);
+            poolTokens[0] = poolToken;
 
-                (rewardTokens, claimableAmounts) = rewardsManager.getAllUserRewards(
-                    poolTokens,
-                    address(this)
-                );
-            }
+            (rewardTokens, claimableAmounts) = rewardsManager.getAllUserRewards(
+                poolTokens,
+                address(this)
+            );
+        }
```

Comment:
I think that we don't need this.

Reply:
@pakim249CAL has a PR for this but I can do it here if you want